scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Split.scala

```diff
@@ -248,20 +250,22 @@ case class Split(
         else ""
       }
     val opt = optimalAt.fold("")(", opt=" + _)
-    s"""$prefix${modExt.mod}:${line.value}(cost=$cost, indents=$indentation, $policy$opt)"""
+    val fileName = Paths.get(fileLine.file.value).toFile().getName()
```

Comment:
i don't think you need any of this, no? isn't FileLine.toString already doing it (i think it might even truncate the .scala)? and if it isn't, I'd change it there first, or add a new method.

Reply:
Yep, it does just that. Thanks!